polkit.yaml

```diff
@@ -0,0 +1,96 @@
+package:
+  name: polkit
+  version: "126"
+  epoch: 0
+  description: "A toolkit for defining and handling authorizations."
+  copyright:
+    - license: LGPL-2.0-or-later
+  dependencies:
+    runtime:
+      - dbus
+      - linux-pam
+      - systemd
+      - wolfi-base
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - dbus-dev
+      - duktape
+      - expat-dev
+      - gettext-dev
+      - glib-dev
+      - glib-gir
+      - gobject-introspection-dev
+      - linux-pam-dev
+      - meson
+      - perl
+      - systemd-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: d627b0d1e1108563658dabe3fb8d2a065e64df10
+      repository: https://github.com/polkit-org/polkit.git
+      tag: ${{package.version}}
+
+  - uses: meson/configure
+    with:
+      opts: |
+        -Dman=true \
+        -Dpam_prefix=/etc/pam.d \
+        -Dpam_module_dir=/usr/lib/security
+
+  - uses: meson/compile
+
+  - uses: meson/install
+
+subpackages:
+  - name: polkit-doc
+    pipeline:
+      - uses: split/manpages
+    dependencies:
+      runtime:
+        - polkit
+    description: polkit manpages
+    test:
+      pipeline:
+        - uses: test/docs
+
+  - name: polkit-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - polkit
+    description: polkit dev
```

Comment:
It looks like this could use a test:

```
2025-02-21T21:20:43Z WARN 🔗 linter "pkgconf" failed on package "polkit-dev": pkgconfig directory found: usr/lib/pkgconfig/polkit-agent-1.pc; suggest: This package provides files in a pkgconfig directory, please add the pkgconf test pipeline
```

Reply:
Thanks, done.